lib/index.ts

```diff
@@ -163,5 +278,5 @@ async function buildDepTreeFromFiles(
   const manifestFileContents = fs.readFileSync(manifestFileFullPath, 'utf-8');
```

Comment:
What if this fails? Can it fail? We don't seem to try/catch much around here.

Reply:
It shouldn't™️ fail, because we are verifying both path via `fs.existsSync` few lines above. If it would fail despite that, it would just throw an error from the function.